src/styles/utils/mixins.scss

```
@@ -0,0 +1,11 @@
+@mixin size-circle-mini {
+  width: 20px;
+  height: 20px;
+}
+
+@mixin size_position-for-hand {
+  position: absolute;
+  left: 49%;
+  bottom: 50%;
```

Your watch looks amazing! But you have some positioning problems/
For example your arrows are not in the center of the block. 
I think it's better to do this: Position the arrows relative to the bottom. And set its block to flex positioning.
```suggestion
 	bottom: 50%;
```